packages/api-gateway-service/src/types.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module '*.graphql';declare module '*.json';
```

Comment:
Also, can you add make this change in the cli microservice? It seems to have come from cli template.

```suggestion
declare module '*.graphql';
declare module '*.json';
```

Reply:
formattted this code and changed this